test/generators/views_generator_test.rb

```diff
@@ -62,6 +62,37 @@ class ViewsGeneratorTest < Rails::Generators::TestCase
     assert_no_directory "app/views/rodauth"
   end
 
+  test "specifying css tailwind" do
+    run_generator %w[--css=tailwind]
+
+    assert_file "app/views/rodauth/login.html.erb"
+    assert_file "app/views/rodauth/logout.html.erb"
+		assert_no_file "app/views/admin/rodauth/logout.html.erb"
+  end
+
+	test "that the tailwind file is being copied, rather than the bootstrap file" do
+		run_generator %w[--css=tailwind logout]
+
+	  assert_file "app/views/rodauth/logout.html.erb", <<~ERB.strip()
+<% content_for :title, rodauth.logout_page_title %>
+<div class="flex justify-center">
+  <%= form_with url: rodauth.logout_path, method: :post, data: { turbo: false } do |form| %>
+    <% if rodauth.features.include?(:active_sessions) %>
+      <div class="py-3">
+        <%= form.check_box rodauth.global_logout_param, id: "global-logout", class: "appearance-none checked:bg-blue-500" %>
+        <%= form.label "global-logout", rodauth.global_logout_label, class: "inline-block text-gray-800" %>
+      </div>
+    <% end %>
+    <div class="py-3">
+      <%= form.submit rodauth.logout_button, class: "px-8 py-2 font-semibold rounded-md flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-blue-600 hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 dark:bg-emerald-400 dark:text-gray-900" %>
+    </div>
+  </div>
+<% end %>
```

Comment:
I think it's enough to just match a Tailwind-specific utility class instead of the whole template, because this way we'll need to copy-paste the template each time we make a change there. And we can include this assertion in the first test, I don't think we need two.

Reply:
I wasn't sure how to test for a utility specific class from the template. So I opted for this approach instead: [3cc210b](https://github.com/janko/rodauth-rails/pull/114/commits/3cc210b8b8a4c1227ce98f732d8742a9438dce9d) - basically extracting the whole template into a private method. I agree it's less than ideal. I can run a simple integration test to test against the utility specific class.

Do not hesitate to revert back to me if required.